src/neuroconv/utils/json_schema.py

```diff
@@ -14,17 +14,25 @@
 
 
 class NWBMetaDataEncoder(json.JSONEncoder):
-    def default(self, o):
+    def default(self, obj):
         # Over-write behaviors for datetime object
-        if isinstance(o, datetime):
-            return o.isoformat()
+        if isinstance(obj, datetime):
+            return obj.isoformat()
 
         # This should transforms numpy generic integers and floats to python floats
-        if isinstance(o, np.generic):
-            return o.item()
+        if isinstance(obj, np.generic):
+            return obj.item()
+
+        # Numpy-versions of various numeric types
+        if isinstance(obj, np.integer):
+            return int(obj)
+        if isinstance(obj, np.floating):
+            return float(obj)
+        if isinstance(obj, np.ndarray):
+            return obj.tolist()
```

Comment:
MEArec header had a bunch of values casted as `numpy` data types, which requires this decoder mapping to serialize